theano/tensor/nnet/abstract_conv.py

```diff
@@ -1507,6 +1526,13 @@ def conv(self, img, kern, mode="valid", dilation=1):
         out_shape = get_conv_output_shape(img.shape, kern.shape,
                                           mode, [1] * self.convdim, dilation)
 
+        out_rows = out_shape[self.convdim]
+        out_cols = out_shape[self.convdim+1]
+
+        if unshared == True:
```

Comment:
Do you have a specific reason for explicitly checking for `== True` here? (See also `unshared == False` below.)